chainerrl/misc/pretrained_models.py

```diff
@@ -0,0 +1,149 @@
+"""This file is a fork from ChainerCV, an MIT-licensed project,
+https://github.com/chainer/chainercv/blob/master/chainercv/utils/download.py
+"""
+
+
+from __future__ import print_function
+from __future__ import division
+from __future__ import unicode_literals
+from __future__ import absolute_import
+from builtins import *  # NOQA
+from future import standard_library
+standard_library.install_aliases()  # NOQA
+
+from distutils.util import strtobool
+import filelock
+import hashlib
+import os
+import shutil
+import tempfile
+import time
+import sys
+import zipfile
+from six.moves.urllib import request
+
+from chainer.dataset.download import get_dataset_directory
+from chainer.dataset.download import get_dataset_root
+
+
+MODELS = {
+    "DQN": ["best", "final"],
+    "IQN": ["best", "final"],
+    "Rainbow": ["best", "final"],
+    "A3C": ["final"],
+    "DDPG": ["best", "final"],
+    "TRPO": ["best", "final"],
+    "PPO": ["final"],
+    "TD3": ["best", "final"],
+    "SAC": ["best", "final"]
+}
+
+url = "https://chainer-assets.preferred.jp/chainerrl/"
```

Comment:
This module-level variable should be renamed as `cached_download` and `download_and_store_model` use a local variable of the same name.

Reply:
What do you mean? This variable should be renamed to `cached_download`? I have a function called `cached_download`. Can I call this `download_url` instead?

Reply:
No, I meant that it should be renamed *because* `cached_download` and `download_and_store_model` use a local variable of the same name.

Reply:
Understood.

Reply:
Addressed.